src/main/resources/templates/ingredients/index.html

```diff
@@ -19,7 +19,7 @@
     </thead>
     <tr th:each="ingredient : ${ingredients}">
         <td th:text ="${ingredient.id}"></td>
-        <td th: text="${ingredient.name}"></td>
+        <td th:text="${ingredient.name}"></td>
```

Comment:
This was causing a really nasty Thymeleaf error that wasn't giving me a very good error description. That coupled with having not used Thymeleaf in a while meant this was the hardest thing for me to track down. Always double-check your `th:` attributes to make sure you haven't introduced extra whitespace, because Thymeleaf will error out.